src/api/app/assets/javascripts/webui/application.js

```diff
@@ -36,6 +36,7 @@
 //= require webui/application/attribute
 //= require webui/application/main
 //= require webui/application/repository_tab
+//= require webui/application/moment
```

Comment:
I see your point. We'll try to find a way to use moment.js rubygem version. Was there anything else you wish tho change in the pull request?